### PR TITLE
Fix gemaps family

### PR DESCRIPTION
**Pull request by frankenjoe**
* Removes LLD deltas
* Adds ``eGeMAPSv02``

Will be released as `v2.0.0`.

### CODE REVIEW

**Comment by hagenw** [comment]
Do we need first a new release of https://github.com/audeering/opensmile?

**Comment by frankenjoe** [comment]
In principle no, since only config files change, but not the binaries. But of course if we release this first it means we can extract feature sets with `pyopensmile` that are not yet available in `opensmile`. But I leave this decision to @chausner-audeering.

**Comment by chausner-audeering** [comment]
Feel free to go ahead with this release.

File: README.rst [hunk 1]
```diff
@@ -18,20 +18,43 @@ Feature sets
...
 * Low-level descriptors (LDD)
-* LLDs with deltas
 * Functionals
 
+For `ComParE 2016`_ a third level is available:
+
+* LLD deltas
+
+.. note:: With v2.0.0 LLD deltas were removed from all sets of the
+    GeMAPS family.
```
Comment (chausner-audeering) [inline README.rst:36]: This sounds a bit like these feature sets really had LLD deltas and we removed them. I suggest to change the wording to say that pre 2.0.0 versions incorrectly output some LLDs as delta LLDs.

**Comment by frankenjoe** [comment]
@chausner-audeering ok! Once you are done with your review please merge or assign back to me.

**Review by chausner-audeering — COMMENTED** [review]
Looks good overall. If not already done or part of the unit tests, we should probably do a manual test that the new feature set indeed works.

**Comment by frankenjoe** [comment]
> Looks good overall. If not already done or part of the unit tests, we should probably do a manual test that the new feature set indeed works.

All feature sets are automatically tested against `SMILExtract`.

File: README.rst (hunk 1 above)
Comment (frankenjoe) [reply README.rst:36]: image

Ok?

Comment (chausner-audeering) [reply README.rst:36]: Yes 👍

**Pull request merged**